src/views/VenuesListWithFilter/FilterList/RoomAmenities/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import ListItem from '@mui/material/ListItem'
+import Checkbox from '@mui/material/Checkbox'
+
+export function RoomAmenities ({ checkedRoomAmenities, setCheckedRoomAmenities }) {
+  const roomAmenitiesFilters = [
```

Comment:
can be declared above the component, we don't need to redeclare it with every rerender